content/en/tracing/api_catalog/_index.md

```diff
@@ -37,105 +33,35 @@ API Catalog combines data from across Datadog to provide opinionated workflows s
-
-In the list table, **sort** by any of the columns by clicking a header. For example, click **P99** to see endpoints with the highest 99th percentile for latency.
-
-The table also shows **team ownership** for the API. This information is inherited from the associated service definition in the [Service Catalog][2]. The service owner owns all of the endpoints connected to the service.
-
-To **filter the list** of endpoints or to search for a particular endpoint of interest, enter a query in the **Search** field. Search by service, path, or any other primary tag. 
-
-Or select a combination of facets on the left, filtering by the owning **service** or the **team** they belong to.
-
-To **scope the data** shown in the table, specify an environment, another primary tag (such as datacenter), and a time frame.
-
-{{< img src="tracing/api_catalog/api-catalog-scope.mp4" alt="Showing how changing scope settings changes the metrics shown in the API Catalog" video="true" >}}
-
-You can [define tags](#group-apis-to-express-a-feature-or-business-logic) to use as facets so you can easily find groups of endpoints you're most interested in.
-
-
-
-## Investigating endpoint details
-
-When exploring an endpoint, getting a higher resolution view of the API endpoint details is helpful.
-
-Click an endpoint in the list to open a details page that shows performance, ownership, and metadata information for the endpoint, collected from various areas of Datadog, into one place. Here you can edit the short name and description and add custom tags. You can also launch deeper investigations of the various types of telemetry using links into other areas of Datadog. 
-
-For example, you can:
-- Look up a specific endpoint by path (for example, `/checkout`) when it shows a high error rate and a high request count.
-- View the Error/Hits graph and correlated Response Code graph to identify issues.
-- Navigate to related telemetry like traces and logs.
+<div class="alert alert-info">See <a href="/tracing/api_catalog/get_started/#key-terminology">Key terminology</a> for some background on concepts that are important to understanding what API Catalog does.</div>
 
-{{< img src="tracing/api_catalog/endpoint-details-pivot-to-traces.mp4" alt="Click into an endpoint to see Errors graph, and click through to related traces to investigate." video="true" >}}
+## Getting started
 
-The performance graphs on the page are initially scoped to the same settings as on the API Catalog page, and you can change those settings on the details page to suit your investigation by using the time frame selector and other scope dropdown menus. 
+If you're already monitoring the performance of your services using APM, API Catalog automatically detects API and endpoints in your instrumented services. 
 
+See [Setting up API Catalog][3] for more information, including how to add APIs and endpoints for services that aren't instrumented.
```

Comment:
maybe replace 'instrumented' here with 'supported for auto discovery' (which is really the case)

Reply:
```suggestion
See [Setting up API Catalog][3] for more information, including how to add APIs and endpoints for services that aren't auto-discoverable.
```